charts/ia-case-api/values.preview.template.yaml

```diff
@@ -64,7 +64,7 @@ java:
       password: hmcts
 
 ia-case-payments-api:
-  enabled: false #set to true if testing against a local branch
+  enabled: true #set to true if testing against a local branch
```

Comment:
If the comment is correct, I presume this needs reverting?

Reply:
Before merge it to master, it will be reverted